apps/site-app/src/containers/Accounts/Transfer/index.tsx

```diff
@@ -60,31 +62,43 @@ export const Transfer = () => {
   const account = AccountMap.useActive();
   const { rates } = useFxRates();
   const rate = weBuyAt(rates);
-  const intl = useIntl();
+
+  const resetForm = () => {
+    setCoinTransfer(null);
+    setResetDefault(Date.now());
+    setForceValidate(false);
+  }
```

Comment:
This code is virtually identical, really needs refactoring to eliminate lotsa code